tests/functional/adapter/materialization/test_snapshots.py

```diff
@@ -97,3 +103,15 @@ def project_config_update(self):
             "seeds": {"+twin_strategy": "prevent"},
             "name": "snapshot_strategy_timestamp",
         }
+
+class TestSnapshotDbtValidToCurrentDremio(BaseSnapshotDbtValidToCurrent):
```

Comment:
Don't we need to override unique_schema and dbt_profile_data functions?

Reply:
I mean should we keep them in TestSnapshotTimestampDremio and TestSnapshotCheckColsDremio classes as well?